README.md

```diff
@@ -14,6 +14,7 @@ yarn add volto-form-block volto-subblocks
 > **Note**: Since version v2.1.2 of this addon, it's required Volto 14.2.0
 >
 > **Note**: Since version v3.0.0 of this addon, it's required Volto >= 16.0.0-alpha.38
+> **Note**: Since version v3.11.0 of this adddon, it's required it's required [collective.volto.formsupport](https://github.com/collective/collective.volto.formsupport) 3.1.4 or higher.
```

Comment:
```suggestion
> - Since version v3.11.0 of this add-on, [`collective.volto.formsupport`](https://github.com/collective/collective.volto.formsupport) 3.1.4 or later is required.
```